src/index.js

```diff
@@ -296,6 +303,51 @@ class Saturn {
       performance.clearResourceTimings()
     }
   }
+
+  async _loadNodes (opts) {
+    let origin = opts.orchURL
+
+    let cachedNodesList
+    if (this.storage) {
+      cachedNodesList = this.storage.get(this.nodesListKey)
```

Comment:
I would name this `cachedNodesListPromise` to make it clear its a pending promise.